pages/11ty/markdown.shortcut.js

```diff
@@ -56,11 +56,17 @@ class MDRenderer {
   }
 
   static resolveLinks(dom, basePath) {
-    dom.querySelectorAll('a[href^="."]').forEach((link) => {
-      const absolutePath = path.join(path.dirname(basePath), link.href).replace(/\\/g, '/');
-      const resultPath = MDRenderer.processRewriteRules(absolutePath);
-      console.log(color.yellow(`Rewrite link "${link.href}" to "${resultPath}"`));
-      link.href = resultPath;
+    dom.querySelectorAll('a[href]').forEach((link) => {
+      if (link.href.startsWith('.')) {
+        const absolutePath = path.join(path.dirname(basePath), link.href).replace(/\\/g, '/');
+        const resultPath = MDRenderer.processRewriteRules(absolutePath);
+        console.log(color.yellow(`Rewrite link "${link.href}" to "${resultPath}"`));
+        link.href = resultPath;
+      }
+      if (link.href.startsWith('http:') || link.href.startsWith('https:')) {
+        link.target = '_blank';
+        link.rel = 'noopener norefferer';
```

Comment:
```suggestion
        link.rel = 'noopener noreferrer';
```

Reply:
By the way, here is one more important point:
It will be better to remove `noreferrer` at all. We have not many cases with any other domains than Github, especially in the current implementation when we have this processor on the MD transformation level.
If we are talking about GitHub domain we SHOULD NOT block any analytics collecting for the GItHub statistics and any SEO counting for esl-ui.com - github.com relationship.

Reply:
I guess value `nofollow` is more critical than `noreferrer`. Value `noreferrer` hides information about the site where you come from. I don't see anything criminal to show this information. But `nofollow` value is used when you want to prevent the search engine from transferring the "search ranking" to another page.